### ISSUE TITLE
[Manage users] Table missing structural information

### ISSUE THREAD

**Issue by svinkle** [CovidShield/portal#6]
## Issue summary

The Manage users `table` is missing `scope` attributes and header cells. Understanding the data and structure in the `table` might be difficult without these extra attributes. The `thead` is also missing a cell to make up the four-column structure. This may lead to a confusing user experience when navigating the table.

<img width="727" alt="Screen Shot 2020-05-19 at 10 41 34 AM" src="https://user-images.githubusercontent.com/1392632/82340563-5e688c00-99bd-11ea-97b0-86b7dcf7d9a2.png">

#### HTML

```html
<table class="index__table">
  <thead class="index__header">
    <tr>
      <th class="index__heading">Username</th>
      <th class="index__heading">Permissions</th>
      <th></th>
    </tr>
  </thead>

  <tbody>
    <tr class="index__row">
      <td class="index__cell">admin</td>
      <td class="index__cell">
        Admin
      </td>
      <td class="index__cell" width="1"></td>
      <td class="index__cell" width="1"></td>
    </tr>

    <!-- … -->
  </tbody>
</table>
```

## Steps to reproduce

1. Go to https://portal.covidshield.app/users with with any supported browser
2. With an active screen reader, navigate through the table content via virtual cursor

### Expected

- Header/row scope is announced for each cell.

### Actual

- Header/row scope is missing.

## Recommendation

Within `thead`:

1. For each `th` element within `thead`, add the `scope="col"` attribute.
2. Add the missing `th` within `thead` to complete the four-column structure.

Within `tbody`:

1. Swap the first `td` element in each row for `th`.
2. For each `th` element within `tbody`, add the `scope="row"` attribute.
3. Remove the obsolete `width` attribute, use CSS for styling.

#### HTML

```html
<table class="index__table">
  <thead class="index__header">
    <tr>
      <th class="index__heading" scope="col">Username</th>
      <th class="index__heading" scope="col">Permissions</th>
      <th></th>
      <th></th>
    </tr>
  </thead>

  <tbody>
    <tr class="index__row">
      <th class="index__cell" scope="row">admin</th>
      <td class="index__cell">
        Admin
      </td>
      <td class="index__cell"></td>
      <td class="index__cell"></td>
    </tr>

    <!-- … -->
  </tbody>
</table>
```

## Specifications

- **Component:** Manage users
- **WCAG Principle:** Perceivable
- **WCAG SC:** 1.3.1 Info and Relationships
- **Severity:** Medium
- **Effort:** Low
